src/f5_tts/eval/eval_infer_batch.py

```diff
@@ -139,17 +139,21 @@ def main():
     model = CFM(
         transformer=model_cls(**model_cfg, text_num_embeds=vocab_size, mel_dim=n_mel_channels),
         mel_spec_kwargs=dict(
-            target_sample_rate=target_sample_rate,
-            n_mel_channels=n_mel_channels,
+            n_fft=n_fft,
             hop_length=hop_length,
+            win_length=win_length,
+            n_mel_channels=n_mel_channels,
+            target_sample_rate=target_sample_rate,
+            extract_backend=extract_backend,
         ),
         odeint_kwargs=dict(
             method=ode_method,
         ),
         vocab_char_map=vocab_char_map,
     ).to(device)
 
-    model = load_checkpoint(model, ckpt_path, device, use_ema=use_ema)
+    dtype = torch.float16 if extract_backend == "vocos" else torch.float32
```

Comment:
need device check
`supports_fp16 = torch.cuda.is_available() and torch.cuda.get_device_properties(device).major >= 6`
or 
`supports_fp16 = device=="cuda" and torch.cuda.get_device_properties(device).major >= 6`